core/phase-register.js

```diff
@@ -8,14 +9,14 @@ function PhaseDefinition(info, impl, defaults) {
 
 PhaseDefinition.prototype.build = function() {
   var defaults = this.defaults;
-  function override(defaults, options) {
+  function override(defaults, experimentOptions) {
     var result = {};
     for (key in defaults) {
-      if (key in options) {
+      if (key in experimentOptions) {
```

Comment:
bit sad to have to rename all these, we should just move the eval to a scope where we can set `options` to the right thing